packages/server/lib/util/ci_provider.js

```diff
@@ -304,6 +313,9 @@ const _providerCiParams = () => {
       'BUILD_URL',
       'BUILD_NUMBER',
       'ghprbPullId',
+      // Jenkins pipeline options
+      'CHANGE_ID',
+      'CHANGE_URL',
```

Comment:
Can we link a reference code to these providers?

Reply:
It does seem like these are specific to the plugins installed on the jenkins instances. 

Looks like the `Multibranch Pipeline` plugin also adds `BRANCH_NAME`: https://www.jenkins.io/doc/book/pipeline/multibranch/#additional-environment-variables

Reply:
I've had Multibranch brought up by users in the recent past, so I would advocate for adding this in if it's not much additional effort.

Reply:
I'll add some comments with links. We've also been using this as a reference for extra env vars:

https://ci.eclipse.org/webtools/env-vars.html/